src/components/datetime/DateTimeController.cpp

```diff
@@ -75,6 +75,22 @@ void DateTime::UpdateTime(uint32_t systickCounter) {
   minute = time.minutes().count();
   second = time.seconds().count();
 
+  if(minute == 0 and not isHourAlreadyNotified) {
+    isHourAlreadyNotified = true;
+    if(systemTask != nullptr)
```

Comment:
Brace style inconsistent with the rest of the code

Reply:
Hopefully my clang-format fixed this now?